### ISSUE TITLE
Unable to Arm/Disarm

### ISSUE THREAD

**Issue by reward72** [wardcraigj/total-connect-client#14]
The integration allows me to see the current status of the panel, but arming/desarming generates an error.  This is what I see in the logs:

```
Error executing service <ServiceCall alarm_control_panel.alarm_disarm: entity_id=['alarm_control_panel.total_connect'], code=>
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.5/site-packages/homeassistant/core.py", line 1002, in _event_to_service_call
    await service_handler.func(service_call)
  File "/srv/homeassistant/lib/python3.5/site-packages/homeassistant/components/alarm_control_panel/__init__.py", line 140, in async_alarm_service_handler
    yield from getattr(alarm, method)(code)
  File "/usr/lib/python3.5/asyncio/futures.py", line 380, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.5/asyncio/tasks.py", line 304, in _wakeup
    future.result()
  File "/usr/lib/python3.5/asyncio/futures.py", line 293, in result
    raise self._exception
  File "/usr/lib/python3.5/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/srv/homeassistant/lib/python3.5/site-packages/homeassistant/components/alarm_control_panel/totalconnect.py", line 94, in alarm_disarm
    self._client.disarm()
  File "/srv/homeassistant/lib/python3.5/site-packages/total_connect_client/TotalConnectClient.py", line 264, in disarm
    deviceId = self.get_security_panel_device_id(location)
  File "/srv/homeassistant/lib/python3.5/site-packages/total_connect_client/TotalConnectClient.py", line 149, in get_security_panel_device_id
    raise Exception('No security panel found')
Exception: No security panel found
```

If that helps, I am able the arm the panel with curl, using these two calls:

```
https://rs.alarmnet.com/TC21API/TC2.asmx/AuthenticateUserLogin?userName=<user>&password=<pass>&ApplicationID=<application_id>&ApplicationVersion=3.0.32
https://rs.alarmnet.com/TC21API/TC2.asmx/ArmSecuritySystem?SessionID=<session_id>&LocationID=<location_id>&DeviceID=<device_id>&ArmType=1&UserCode=-1
```

**Comment by wardcraigj** [comment]
Thanks! What value do you use for the Device ID?

**Comment by reward72** [comment]
642641. I don't remember where I found that number.

**Comment by wardcraigj** [comment]
ok, I'll take a poke at this when I get home from work tonight. If you happen to come across a DeviceName for that panel (it will probably be in the same chunk of info that you originally got the DeviceId), let me know.

**Comment by reward72** [comment]
I remember now that it's from the API function GetSessionDetails.  The DeviceName is "Lynx Touch-WiFi"

**Comment by wardcraigj** [comment]
Fix is merged in #15 and uploaded to Pypi, version still needs to be bumped in HASS
